lib/rules/selector-list-comma-newline-after/index.js

```diff
@@ -53,7 +53,7 @@ const rule = (primary, _secondaryOptions, context) => {
 					functionArguments: 'skip',
 				},
 				(match) => {
-					const nextChars = selector.substr(match.endIndex, selector.length - match.endIndex);
+					const nextChars = selector.slice(match.endIndex);
```

Comment:
[note] Since `selector.length - match.endIndex` means to extract a sub-string from `match.endIndex` to the end of `selector`, it can be replaced by omitting the second argument.